Neos.Flow/Classes/ResourceManagement/Target/SymlinkedFolderResourceObject.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neos\Flow\ResourceManagement\Target;
+
+use Neos\Flow\ResourceManagement\ResourceMetaDataInterface;
+
+/**
+ * @internal only for the callback {@see TargetInterface::onPublish()}
+ */
+class SymlinkedFolderResourceObject implements ResourceMetaDataInterface
```

Comment:
This might need some discussion and feel odd to you.
Its a necessity of the FileSystemSymlinkTarget